### PR TITLE
feat: add hasVideo label

### PR DESCRIPTION
**Pull request by colincclark**
https://nidigitalsolutions.jira.com/browse/REPLAT-1619

The `hasVideo` label comes from tpa and indicates whether an article has a video as the lead asset, or even included in the article content (inline). We now use a video label instead of a normal label component in this case in the following:

- article pages
- article list pages - the first article in the fixture is marked as having a video
- related articles - (only the Standard Slice related article fixture that already had a video)

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/5efd8ba7edb94674b63210af55e3715b6ba6f5d9/index.html

File: packages/article/__tests__/android/__snapshots__/article-with-style.android.test.js.snap [hunk 1]
```diff
@@ -363,16 +363,6 @@ exports[`3. an article with a video as the lead asset 1`] = `
           }
         }
       >
-        <View
```
Comment (craigbilner) [inline packages/article/__tests__/android/__snapshots__/article-with-style.android.test.js.snap]: should we not just keep these snaps the same?

File: packages/article/src/article-header/article-header.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ const { style: ViewStylePropTypes } = ViewPropTypes;
...
+  hasVideo,
   headline,
-  isVideo,
   label,
   standfirst,
   style
 }) => (
   <View style={[...style]}>
-    <HeaderLabel isVideo={isVideo} label={label} />
+    <HeaderLabel hasVideo={hasVideo} label={label} />
```
Comment (craigbilner) [inline packages/article/src/article-header/article-header.js]: did `isVideo`  not make more sense here?

Comment (colincclark) [reply packages/article/src/article-header/article-header.js]: hmmm....changed

File: packages/article/__tests__/android/__snapshots__/article-with-style.android.test.js.snap (hunk 1 above)
Comment (colincclark) [reply packages/article/__tests__/android/__snapshots__/article-with-style.android.test.js.snap]: I have no idea why the label has been removed here?!?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/58136b06ba6ba0570228cd40bfacaf66b5191f53/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b99467d6c4bd75b3bfd21eaac1acec45df667449/index.html

**Pull request merged**